substrate-node/runtime/src/lib.rs

```diff
@@ -444,6 +453,35 @@ impl pallet_scheduler::Config for Runtime {
     type WeightInfo = ();
 }
 
+parameter_types! {
+	pub const CouncilMotionDuration: BlockNumber = 3 * MINUTES;
+	pub const CouncilMaxProposals: u32 = 100;
+	pub const CouncilMaxMembers: u32 = 100;
```

Comment:
This needs to be revised. At least the motion duration will need to be increased.

Reply:
What do you suggest the motionduration should be?

Reply:
better a bit too long than too short, 7days?